packages/hub/src/app/groups/[slug]/members/GroupReusableInviteSection.tsx

```diff
@@ -0,0 +1,144 @@
+import { FC, useEffect, useMemo, useState } from "react";
+import Skeleton from "react-loading-skeleton";
+import { useFragment } from "react-relay";
+import { graphql } from "relay-runtime";
+
+import { ClipboardCopyIcon, TextTooltip, useToast } from "@quri/ui";
+
+import { GroupReusableInviteSection$key } from "@/__generated__/GroupReusableInviteSection.graphql";
+import { GroupReusableInviteSection_CreateMutation } from "@/__generated__/GroupReusableInviteSection_CreateMutation.graphql";
+import { GroupReusableInviteSection_DeleteMutation } from "@/__generated__/GroupReusableInviteSection_DeleteMutation.graphql";
+import { H2 } from "@/components/ui/Headers";
+import { MutationButton } from "@/components/ui/MutationButton";
+import { groupInviteLink } from "@/routes";
+
+type Props = {
+  groupRef: GroupReusableInviteSection$key;
+};
+
+export const GroupReusableInviteSection: FC<Props> = ({ groupRef }) => {
+  const group = useFragment(
+    graphql`
+      fragment GroupReusableInviteSection on Group {
+        id
+        slug
+        reusableInviteToken
+      }
+    `,
+    groupRef
+  );
+
+  const toast = useToast();
+
+  // Necessary for SSR and to avoid hydration errors
+  const [origin, setOrigin] = useState("");
+  useEffect(() => setOrigin(window.location.origin), []);
```

Comment:
This was annoying to code, there's no way to access hub domain on server, so I had to create the link on the frontend after initial render.

We could configure the domain in env variables to simplify this, but that would complicate our CI (especially for preview domains). There's also [VERCEL_URL](https://vercel.com/docs/projects/environment-variables/system-environment-variables#system-environment-variables), but in my experience it's unreliable.